firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -6667,6 +6667,19 @@ extension String {
         tableName: nil,
         value: nil,
         comment: "See http://mzl.la/1Qtkf0j")
+
+    public struct RemoteTabPanel {
+        public static let CloseTab_SendingNotification_title = MZLocalizedString(
+            key: "CloseTab.SendingNotification.title.v133",
+            tableName: "RemoteTabPanel",
+            value: "Tab closed",
+            comment: "Text in notification after sending close remote tab command.")
```

Comment:
I took a look and we currently have this string for "Tab closed":

```
public static let SingleTabTitle = MZLocalizedString(
    key: "CloseTabsToast.SingleTabTitle.v113",
    tableName: "TabsTray",
    value: "Tab Closed",
    comment: "When the user closes an individual tab in the tab tray, a popup will appear informing them the tab was closed. This is the text for the popup.")

```
Does this sound like it will meet your needs? We may not need to add this string.